internal/provider/resource_aws_transit_gateway_attachment.go

```diff
@@ -322,3 +327,33 @@ func setTransitGatewayAttachmentResourceData(d *schema.ResourceData, tgwAtt *net
 
 	return nil
 }
+
+// resourceAwsTransitGatewayAttachmentImport implements the logic necessary to
+// import an un-tracked (by Terraform) Network peering resource into Terraform
+// state.
+func resourceAwsTransitGatewayAttachmentImport(ctx context.Context, d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	client := meta.(*clients.Client)
+
+	idParts := strings.SplitN(d.Id(), ":", 2)
+	if len(idParts) != 2 || idParts[0] == "" || idParts[1] == "" {
+		return nil, fmt.Errorf("unexpected format of ID (%q), expected {hvn_id}:{transit_gateway_attachment_id}", d.Id())
+	}
+	hvnID := idParts[0]
+	tgwAttID := idParts[1]
+	loc := &sharedmodels.HashicorpCloudLocationLocation{
+		ProjectID: client.Config.ProjectID,
```

Comment:
aren't we also need an org-id here?

Reply:
This location is used to create the link format we use for ids (`/project/123/hashicorp.network.tgw-attachment/456`), so we only actually need the project id. The link format used to include more information, but since it is now only the project id that gets used, I'll make a note to consider simplifying this across our resources 🙏 .

Reply:
Ticket created: https://hashicorp.atlassian.net/browse/HCPE-915